rudder-agent/SOURCES/patches/fusioninventory/20670-windows-inventory-hooks.patch

```diff
@@ -0,0 +1,73 @@
+diff --git a/lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm b/lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm
+index 94187b2..6d7be6d 100644
+--- a/lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm
++++ b/lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm
+@@ -78,15 +78,24 @@ sub _getCustomProperties {
+         closedir(DIR);
+         while (my $file = shift @ordered_script_list) {
+             my $script_file = $custom_properties_dir . "/" . $file;
++            my $properties;
+             if (-f $script_file) {
+                 next if ($file =~ m/^\./);
+-                # Ignore non-executable file, or folders
```

Comment:
shouldn'y we keep this on linux ?

Reply:
It is kept, but moved to the `else` section which is dedicated for non-windows agents.
See https://github.com/Normation/rudder-packages/pull/2569/files#diff-82fb8f30c785b82e012736e775301649e2374483bf82eff6df02d6ca4018821bR30